CSharpSourceCode/AttributeDataSystem/StaticAttribute.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.SaveSystem;
+
+namespace TOW_Core.AttributeDataSystem
+{
+    /// <summary>
+    /// Contains Tow data of single unit or character. For attributes affecting whole Party use a PartyAttribute
+    /// </summary>
+    public class StaticAttribute
+    {
+        //dummy data types
+        [SaveableField(0)] public string race;  //mostly hardcoded currently needs to be assigned via xml or similar also counts for many other values in this class
+        [SaveableField(1)] public string culture; // eg. undead, Vampire, Human
+        [SaveableField(2)]public bool MagicUser;   
```

Comment:
Wouldnt IsMagicUser be a better name for bool?

Reply:
agree